src/resources/Notifications/Subscriptions/SubscriptionsInterfaces.ts

```diff
@@ -0,0 +1,53 @@
+import {SubscriptionFrequencyEnum, SubscriptionStatusHealthIndicatorEnum, SubscriptionTypeEnum} from '../../Enums';
+
+export interface EditSubscriptionModel {
+    description: string;
```

Comment:
I'm not sure if all these fields are required? IIRC normally `description` is optional in the POST calls :thinking:

Reply:
No precision whatsoever in the swagger or in the definitions if the network tab.

Reply:
> Those calls must be unit tested

Yes... I keep forgetting that.

Reply:
You are probably right though, so I made the description optionnal :+1: